pallets/dao/src/tests.rs

```diff
@@ -515,26 +615,39 @@ fn user_can_view_organization_it_belongs_to_member_of() {
 fn user_can_be_removed_from_organization_it_belongs_to_member_of() {
 	new_test_ext().execute_with(|| {
 
-		// Create Static Organization name
-		const ORG_NAME1: &'static [u8] = &[7];
-		const ORG_NAME2: &'static [u8] = &[8];
+		// Create Static Organization name, description, vision
+		const ORG_NAME1: &'static [u8] = &[10];
```

Comment:
Instead of constantly redefining these, can we make them global? It will make the tests more readable..